circuits/cpp/barretenberg/cpp/src/barretenberg/common/fuzzer.hpp

```diff
@@ -68,7 +69,9 @@ class FastRandom {
     FastRandom(uint32_t seed) { reseed(seed); }
     uint32_t next()
     {
-        state = static_cast<uint32_t>((uint64_t(state) * uint64_t(363364578) + uint64_t(537)) % uint64_t(3758096939));
+        state = static_cast<uint32_t>(
```

Comment:
This is not really readable due to everything being on one line and the magic numbers -- Its in the fuzzer so we can avoid

Reply:
yeah given its fuzzer.hpp I think it's ok to leave as-is